pkg/liqoctl/install/helm.go

```diff
@@ -61,6 +61,15 @@ func initLiqoRepo(helmClient helm.Client) error {
 		return err
 	}
 
+	capsuleRepo := repo.Entry{
+		URL:  "https://clastix.github.io/charts",
+		Name: "capsule",
+	}
```

Comment:
Given these values are hard-coded both here and in the `charts.yaml` file, I would add a comment to link them and warn to update both if it will be required in the future.